### PR TITLE
Update "What's New in DevTools 128"

### PR DESCRIPTION
**Pull request by leahmsft**
Rendered article for review:
* **What's New in DevTools (Microsoft Edge 128)**
   * https://review.learn.microsoft.com/en-us/microsoft-edge/devtools-guide-chromium/whats-new/2024/08/devtools-128?branch=pr-en-us-3256
      * Heap snapshot improvements
      * View WebSocket message activities in the Performance tool
   * [Before/Live](https://learn.microsoft.com/microsoft-edge/devtools-guide-chromium/whats-new/2024/08/devtools-128)

This PR adds two new entries.

AB#53506860

### CODE REVIEW

File: microsoft-edge/devtools-guide-chromium/whats-new/2024/08/devtools-128.md [hunk 1]
```diff
@@ -14,11 +14,11 @@ ms.date: 08/22/2024
 
 
 <!-- ====================================================================== -->
-## Updated labels for selector stats
+## Updated labels for CSS selector stats
 
 <!-- Subtitle: Use the "Enable CSS selector stats" setting instead of the "Enable advanced paint instrumentation (slow)" to capture CSS selector statistics for Recalculate Style events -->
 
-In the **Performance** tool, the duplicate **Selector Stats** label has been removed from the `Recalculate Style` event details.  Selector stats can be enabled by using the dedicated setting for it.
+In the **Performance** tool, the duplicate **Selector Stats** label has been removed from the `Recalculate Style` event details.  Selector stats can be enabled by using the dedicated setting<!-- todo: what is the ui path? eg: the Foo checkbox --> for it.
```
Comment (mikehoffms) [inline microsoft-edge/devtools-guide-chromium/whats-new/2024/08/devtools-128.md:21]: where/what specifically is "the dedicated setting for it"?  
@leahmsft

File: microsoft-edge/devtools-guide-chromium/whats-new/2024/08/devtools-128.md [hunk 2]
```diff
@@ -45,6 +48,36 @@ The top of each context menu now shows the title again:
...
+<!-- ====================================================================== -->
+## Heap snapshot improvements
+
+<!-- Subtitle: Heap snapshots are generated faster, HTML elements are grouped by tag name, shallow sizes show fewer internal objects, and numeric values are always shown. -->
+
+In the **Memory** tool, there are several improvements for heap snapshots:
+
+* Generating heap snapshots is now faster.
+
+* HTML elements are grouped by tag name.<!-- todo: by element name? -->
```
Comment (mikehoffms) [inline microsoft-edge/devtools-guide-chromium/whats-new/2024/08/devtools-128.md:60]:
```suggestion
* HTML elements are grouped by element name.
```
alts:
* by element name
* by the starting tab of the element
* by the element name
* by the tag name

somehow i find the existing wording unclear, need to see UI to clarify wording

File: microsoft-edge/devtools-guide-chromium/whats-new/2024/08/devtools-128.md [hunk 3]
```diff
@@ -45,6 +48,36 @@ The top of each context menu now shows the title again:
...
+
+* Shallow sizes now match JavaScript semantics better, by showing fewer internal objects, unless the **Expose internals (includes additional implementation-specific details)** setting is enabled<!--todo: checkbox is selected?  where/what is this UI control?  not found after turned experiment on, and then created snapshot --> when the [Show option to expose internals in heap snapshots](../../../experimental-features/index.md#show-option-to-expose-internals-in-heap-snapshots) experiment is turned on.
```
Comment (mikehoffms) [inline microsoft-edge/devtools-guide-chromium/whats-new/2024/08/devtools-128.md:62]: "unless the **Expose internals (includes additional implementation-specific details)** setting** is enabled ...

Does that mean when the **Expose internals (includes additional implementation-specific details)** checkbox is selected?  where/what is this UI control?  such a UI control is not found after turned the experiment on, and then created snapshot.
@leahmsft

File: microsoft-edge/devtools-guide-chromium/whats-new/2024/08/devtools-128.md [hunk 4]
```diff
@@ -45,6 +48,36 @@ The top of each context menu now shows the title again:
...
+
+* The **Include numerical values in capture** setting<!-- todo: where was this setting in the UI?  was it a checkbox? --> has been removed, and heap snapshots now always show numeric values.
```
Comment (mikehoffms) [inline microsoft-edge/devtools-guide-chromium/whats-new/2024/08/devtools-128.md:64]: Where was the **Include numerical values in capture** setting in the v127- UI?  was it a checkbox?

File: microsoft-edge/devtools-guide-chromium/whats-new/2024/08/devtools-128.md (hunk 2 above)
Comment (leahmsft) [reply microsoft-edge/devtools-guide-chromium/whats-new/2024/08/devtools-128.md:60]: I meant HTML tags. ex: , <div>, <body>, etc.

File: microsoft-edge/devtools-guide-chromium/whats-new/2024/08/devtools-128.md (hunk 1 above)
Comment (leahmsft) [reply microsoft-edge/devtools-guide-chromium/whats-new/2024/08/devtools-128.md:21]: the checkbox is named "Enable CSS selector stats (slow)" in the Perf panel

File: microsoft-edge/devtools-guide-chromium/whats-new/2024/08/devtools-128.md (hunk 3 above)
Comment (leahmsft) [reply microsoft-edge/devtools-guide-chromium/whats-new/2024/08/devtools-128.md:62]: Yes, it is referring to the checkbox below Heap snapshot (before you take the snapshot): 
image

File: microsoft-edge/devtools-guide-chromium/whats-new/2024/08/devtools-128.md (hunk 4 above)
Comment (leahmsft) [reply microsoft-edge/devtools-guide-chromium/whats-new/2024/08/devtools-128.md:64]: Yes, another checkbox before taking the heap snapshot: 
image

File: microsoft-edge/devtools-guide-chromium/whats-new/2024/08/devtools-128.md (hunk 1 above)
Comment (mikehoffms) [reply microsoft-edge/devtools-guide-chromium/whats-new/2024/08/devtools-128.md:21]: result:
```
In the **Performance** tool, the duplicate **Selector Stats** label 
has been removed from the `Recalculate Style` event details.  To 
enable selector stats, use the **Enable CSS selector stats (slow)** 
checkbox in the **Performance** tool.
```

File: microsoft-edge/devtools-guide-chromium/whats-new/2024/08/devtools-128.md (hunk 2 above)
Comment (mikehoffms) [reply microsoft-edge/devtools-guide-chromium/whats-new/2024/08/devtools-128.md:60]: result:
```
* HTML elements are grouped by tag name, 
such as `<body>`, `<div>`, or `<img>`.
```

File: microsoft-edge/devtools-guide-chromium/whats-new/2024/08/devtools-128.md (hunk 3 above)
Comment (mikehoffms) [reply microsoft-edge/devtools-guide-chromium/whats-new/2024/08/devtools-128.md:62]: result:
```
Shallow sizes now match JavaScript semantics better, by showing 
fewer internal objects (unless you select 
the [Show option to expose internals in heap snapshots](../../../experimental-features/index.md#show-option-to-expose-internals-in-heap-snapshots) 
experiment checkbox and then, before taking a snapshot, select 
the **Expose internals (includes additional implementation-specific details)** 
checkbox).
```

File: microsoft-edge/devtools-guide-chromium/whats-new/2024/08/devtools-128.md (hunk 4 above)
Comment (mikehoffms) [reply microsoft-edge/devtools-guide-chromium/whats-new/2024/08/devtools-128.md:64]: result:
```
The **Include numerical values in capture** checkbox (shown 
before you take a heap snapshot) has been removed, and heap 
snapshots now always show numeric values.
```

**Review by mikehoffms — APPROVED** [review]
Approved the PR per Writer/Editor review, after making changes.

**Pull request merged**